dashboard/test/ui/README.md

```diff
@@ -46,7 +46,10 @@ You can find the values for these settings in your saucelabs account settings (`
 If you want to run tests on saucelabs against localhost you need to set up your tunnel:
 
 1. Login to Saucelabs and download the [tunnel](https://app.saucelabs.com/tunnels)
-2. Start the tunnel via `~/bin/sc/ -u $SAUCELABS_USERNAME -k SAUCELABS_ACCESS_KEY`. Notes: The link above has an example command line with your credentials that you can copy. `~/bin/sc/` should be replaced with the path that points to the tunnel software downloaded in step 1.
+2. Start the tunnel via `~/bin/sc/ -u $SAUCELABS_USERNAME -k SAUCELABS_ACCESS_KEY`
+   - Notes: The link above has an example command line with your credentials that you can copy.
+     - `bin/sc` should be replaced with the path that points to the tunnel software downloaded in step 1.
```

Comment:
```suggestion
     - `~bin/sc/` should be replaced with the path that points to the tunnel software downloaded in step 1.
```

Reply:
the example command is now `bin/sc`, not `~bin/sc/`; I updated this to match